src/hooks.server.ts

```diff
@@ -9,19 +9,21 @@ import { addYears } from "date-fns";
 import { collections } from "$lib/server/database";
 import { base } from "$app/paths";
 import { requiresUser } from "$lib/server/auth";
+import { sequence } from "@sveltejs/kit/hooks";
 
-export const handle: Handle = async ({ event, resolve }) => {
+const authorization: Handle = async ({ event, resolve }) => {
 	const token = event.cookies.get(COOKIE_NAME);
 
 	event.locals.sessionId = token || crypto.randomUUID();
 
 	const user = await collections.users.findOne({ sessionId: event.locals.sessionId });
 
-	if (user) {
+	if (user?.hfUserId) {
 		event.locals.userId = user._id;
 	}
 
 	if (
+		!event.url.pathname.startsWith(`${base}/login`) &&
```

Comment:
The redirect on `/login` can probably be moved to a `GET`, avoiding this change

Reply:
SvelteKit doc says:
> Actions always use POST requests, since GET requests should never have side-effects.

So I suppose if we want to keep using an action form this is necessary?

Reply:
but we don't need to use an action form for it, it can just be a link to `/login`